### ISSUE TITLE
Validate: Upgrade *_1-keys

### ISSUE THREAD

**Issue by tordans** [openstreetmap/iD#6387]
There are a lot of keys in OSM like `<something>_1`.

I am not sure about the history of those, but I believe older versions of iD(?) would count up keys that where added twice(?) like this?

As far as I know, those keys are more or less broken data.

**Idea: The validator could help update them.**

I see two cases:

**a. A node has a `<something>_1`-key, but not a `<something>`-key.**
In this case, the validator could offer to rename the key.
The data stays the same.

I am not sure if this is really a case; I think it could only exist if the `<something>` was removed, but `<something>_1` stayed for some reason.

Is there a way to check how many of those situations are there?

**b. A node has a `<something>_1`-key _and_ a `<something>`-key.**
In those cases, the validator could point out the error with two quick actions:
1. Keep the `<something>_1`-value and rename the key.
1. Remove the `<something>_1`-key and -value.

**Counts**
- There are more than 1 million of those keys in taginfo: https://taginfo.openstreetmap.org/search?q=_1
- And another 400k+ for `<something>_2` https://taginfo.openstreetmap.org/search?q=_2
- And even 80k+ for `<something>_3` https://taginfo.openstreetmap.org/search?q=_3
- 13k+ for `<something>_4` https://taginfo.openstreetmap.org/search?q=_4

The usefulness of the search end there, for `_5` there are mostly false positives in https://taginfo.openstreetmap.org/search?q=_5.

Are there cases, where keys with `_1`, `_2`, `_3`, `_4` are "by design"?

**Comment by BjornRasmussen** [comment]
> a. A node has a <something>_1-key, but not a <something>-key.
> In this case, the validator could offer to rename the key.
> The data stays the same.

Hmm.... If an object has this combination, it obviously was changed from another feature type at one point, so the *_1 tag is most likely invalid.

**Comment by bhousel** [comment]
oh yeah the historic issue about these is #2896
(There is a lot of iD hate on that issue).

tl;dr:

- The practice of suffixing keys with numbers when there was a key ambiguity predates iD.  We didn't start this.
- For a while, iD would add a number suffix if a user tried adding the same key twice in the raw tag editor.  We did this to avoid overwriting the existing data.  We don't do this anymore, instead the raw tag editor will switch focus to the other field.

I would interpret the existence of `name` and `name_1` as "I don't know what the correct value should be but I have a slight preference for `name`".

I really don't want to touch this issue anymore.
